src/handlers/DibiHandler.php

```diff
@@ -13,17 +14,17 @@
 class DibiHandler implements IQuery
 {
 
-	/** @var DibiEvent */
+	/** @var Event */
```

Comment:
This will only work with dibi `>=3.0.0`. If support for 3 is added, it should probably not drop support for version 2.

This implementation achieves something similar  https://github.com/nextras/migrations/commit/973bded27ed4d68ae533b47492e01196b5e5c661